src/imio/smartweb/core/contents/sections/news/view.py

```
@@ -34,37 +34,27 @@ def get_news_folders_uids_and_title_from_entity(self, entity_uid):
-        #         target_language=current_lang,
-        #     )
-        # specific_related_newsitems = self.context.specific_related_newsitems
-        # if specific_related_newsitems:
-        #     selected_item = "&".join(
-        #         [f"UID={newsitem_uid}" for newsitem_uid in specific_related_newsitems]
-        #     )
-
+        uids, data = self._get_news_folders_uids_and_title_from_entity(entity_uid)
         selected_item = f"selected_news_folders={self.context.related_news}"
+        if self.context.related_news not in uids:
+            item = [k for k, v in data.items() if "administration" in v.lower()][0]
```

Ruff (0.15.2)

[warning] Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)